extensions/src/ACRE2TS/TsCallbacks_static.cpp

```diff
@@ -42,19 +44,24 @@ void ts3plugin_infoData(uint64 serverConnectionHandlerID, uint64 id, enum Plugin
             break;
         case PLUGIN_CLIENT:
             if (ts3Functions.getClientVariableAsString(serverConnectionHandlerID, (anyID)id, CLIENT_META_DATA, &metaData) == ERROR_ok) {
-                if (!metaData) { 
-                    noAcre = TRUE;
-                }
-                int metaDataLength = strlen(metaData);
-                if (metaDataLength < 3 || metaDataLength > (INFODATA_BUFSIZE - 2)) {
+                if (!metaData) {
                     noAcre = TRUE;
+                } else {
+                    sharedMsg = metaData;
+                    size_t start_pos = sharedMsg.find(START_DATA);
+                    size_t end_pos = sharedMsg.find(END_DATA);
+                    if (start_pos == std::string::npos || end_pos == std::string::npos) {
+                        noAcre = TRUE;
```

Comment:
Do not use upper case in here. Use `true` or `false`. This is an ongoing cleanup process.